Sources/Quick/DSL/World+DSL.swift

```diff
@@ -134,4 +158,18 @@ extension World {
     internal func pending(description: String, closure: () -> ()) {
         print("Pending: \(description)")
     }
+
+    private var currentPhase: String {
+        if currentExampleMetadata != nil {
+            if beforesCurrentlyExecuting {
+                return "beforeEach"
+            } else if aftersCurrentlyExecuting {
+                return "afterEach"
+            }
+
+            return "it"
+        }
+
+        return ""
```

Comment:
I'm curious how other people feel about this. From the new ```currentPhase``` function, it's technically possible to return an empty string if the function is misused. I don't feel super great about this, but I didn't feel super great about the alternatives either --

1) Return some legitimate value as a default value, like "it". 
2) Throw an error -- seems very heavyweight. 

Thoughts?